data_utils/ds_addmult_mod10_helpers.py

```diff
@@ -105,7 +106,9 @@ def build_dataset_addmult_mod10(
         # sample unique subtrees
         ds_uniques = set(dataset.unique('tree_sig'))
         n_subtrees = int(len(ds_uniques) * hold_out_p_subtrees)
-        held_out_subtrees = set(random.sample(ds_uniques, n_subtrees))
+        # This is not reproducible
+        # held_out_subtrees = set(random.sample(ds_uniques, n_subtrees))
+        held_out_subtrees = set(list(ds_uniques)[:n_subtrees])
```

Comment:
This is fine with me, but won't this require running a full epoch to get a proper understanding of the performance of network if the distribution of generated samples is uneven? Might want to check with Derek or highlight this one so that everyone is onboard.

Reply:
That's odd, I think Github glitched with the merge. This is different on my local branch. Will resolve!